lispusers/GREP

```diff
@@ -1,32 +1,34 @@
 (DEFINE-FILE-INFO PACKAGE "INTERLISP" READTABLE "INTERLISP" BASE 10)
 
-(FILECREATED "19-Jul-2022 22:26:08" 
-{DSK}<Users>kaplan>Local>medley3.5>working-medley>lispusers>GREP.;11 4725   
+(FILECREATED "21-Jan-2024 09:29:13" {WMEDLEY}<lispusers>GREP.;24 6676   
 
-      :CHANGES-TO (FNS DOGREP)
+      :EDIT-BY rmk
 
-      :PREVIOUS-DATE "26-Jun-2022 14:36:21" 
-{DSK}<Users>kaplan>Local>medley3.5>working-medley>lispusers>GREP.;9)
+      :CHANGES-TO (FNS LISPSOURCEOUTCHARFN)
 
+      :PREVIOUS-DATE "20-Jan-2024 14:14:15" {WMEDLEY}<lispusers>GREP.;23)
 
-(* ; "
-Copyright (c) 1984-1986 by Xerox Corporation.
-")
 
 (PRETTYCOMPRINT GREPCOMS)
 
-(RPAQQ GREPCOMS ((FNS DOGREP GREP PHONE)
-                 (INITVARS (PHONELISTFILES))))
+(RPAQQ GREPCOMS
+       [(FNS DOGREP GREP LISPSOURCEOUTCHARFN TGREP)
+        (P (MOVD? 'NILL 'TEDIT.FORMATTEDFILEP))
+        (DECLARE%: EVAL@COMPILE DONTCOPY (FILES (FROM VALUEOF (MEDLEYDIR "loadups"))
```

Comment:
could we load exports.all more consistently, or move the functionality into tedit or ...

Reply:
Yeah, not a big fan of `(FROM VALUEOF (MEDLEYDIR "loadups"))` embedded in the COMS.